pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -277,20 +279,24 @@ func (l *podNIC) PlugPhase2(domain *api.Domain) error {
 		return nil
 	}
 
-	domainIface, err := l.cachedDomainInterface()
-	if err != nil {
-		log.Log.Reason(err).Critical("failed to load cached interface configuration")
-	}
-	if domainIface == nil {
-		log.Log.Reason(err).Critical("cached interface configuration doesn't exist")
+	libvirtIfaceSpec := api.Interface{}
```

Comment:
why is this change needed and not just pass `*domainIface`, like it was before?
from what I see nothing is done with `libvirtIfaceSpec` except passing it's value to `decorateConfig` func?

Reply:
It **is** needed. Please note I've changed the code so that slirp **does not** do anything at all on phase#1 - not even cache the interfaces - as it should always have done.

This - in turn - makes it perfectly fine for slirp binding types to not recover the domain interface from the cache on phase#2.

Reply:
ohh right.
Thanks for explaining.

Reply:
No problem, good question.